test/test-tcp-exportimport.c

```diff
@@ -0,0 +1,237 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
```suggestion
/* Copyright libuv contributors. All rights reserved.
```
New files should use the new copyright (but is this file new or copied over from the old implementation? It looks vaguely familiar.)